test/plugins/scenarios/gozero/config/excepted.yml

```diff
@@ -0,0 +1,136 @@
+    segmentSize: ge 1
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: GET:/user/info
+            parentSpanId: -1
+            spanId: 1
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5023
+            isError: false
+            spanType: Entry
+            peer: ''
+            skipAnalysis: false
+            tags:
+              - {key: http.method, value: GET}
+              - {key: url, value: 'localhost:8888/user/info'}
+              - {key: http.params, value: not null}
+              - {key: status_code, value: '200'}
+            refs:
+              - {parentEndpoint: 'GET:/user/info?id=1', networkAddress: 'localhost:8080', refType: CrossProcess,
+                 parentSpanId: 1, parentTraceSegmentId: not null, parentServiceInstance: not null,
+                 parentService: gozero, traceId: not null}
+          - operationName: logx.info
+            parentSpanId: 1
+            spanId: 2
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5023
+            isError: false
+            spanType: Local
+            tags:
+              - {key: content, value: 'user api find: &{Id:0 Uuid:}'}
+          - operationName: logx.error
+            parentSpanId: 1
+            spanId: 3
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 5023
+            isError: false
+            spanType: Local
+            tags:
+              - {key: content, value: 'test error log'}
+          - operationName: user.User.UserFind
+            parentSpanId: 1
+            spanId: 4
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 23
+            isError: false
+            spanType: Exit
+            tags:
+              - { key: url, value: not null }
+              - { key: rpc.type, value: Unary }
+          - operationName: user.User.UserFind/Client/Request/SendMsg
+            parentSpanId: 4
+            spanId: 5
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 23
+            isError: false
+            spanType: Local
+            tags:
+              - { key: url, value: not null }
+          - operationName: user.User.UserFind/Client/Response/RecvMsg
+            parentSpanId: 4
+            spanId: 6
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 23
+            isError: false
+            spanType: Local
+            tags:
+              - { key: url, value: not null }
+          - operationName: user.User.UserFind
+            parentSpanId: 4
+            spanId: 7
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 23
+            isError: false
+            spanType: Entry
+            tags:
+              - { key: url, value: not null }
+              - { key: rpc.type, value: Unary }
+          - operationName: /user.User/UserFind
```

Comment:
I think zrpc should be a framework similar to grpc, mainly used for communication between multiple processes, right? Then why is the RPC-related span still local? As I understand it, there should be entry and exit spans.

Reply:
The method enhanced by zrpc is to add a unary interceptor. The underlying service is still a gRPC service. The purpose of adding this unary interceptor is to extend the RPC service provided by go-zero, such as logging parameters, responses, errors, etc. Currently, due to proxy restrictions, additional packages cannot be introduced, so the span can be considered local. Additionally, can you explain the issue of not being able to introduce additional packages in the interceptor? Otherwise, it will be impossible to complete some more distinctive tag reporting.

Reply:
For the additional package import, please read this document to know the [implementation key principles](https://skywalking.apache.org/docs/skywalking-go/next/en/concepts-and-designs/key-principles/).

Reply:
If all of spans still in one single segment(only create local rather entry and exit span), we cannot building the topology, because OAP don't know which one is exit span to communicate with other application.